lib/kamal/commands/proxy/exec.rb

```diff
@@ -1,12 +1,12 @@
-module Kamal::Commands::App::Proxy
+module Kamal::Commands::Proxy::Exec
```

Comment:
I don't think we gain that much from this deduplication. Let's just keep `Kamal::Commands::App::Proxy` as it is and add `Kamal::Commands::Accessory::Proxy` for the accessory.

Maybe there's a case for deduplication between `App` and `Accessories` in general but it should be tackled in one go rather than piecemeal.